app/extensions.js

```diff
@@ -137,6 +137,12 @@ let generateBraveManifest = () => {
     cspDirectives['style-src'] = '\'self\' \'unsafe-inline\' http://' + devServer
   }
 
+  // TODO:
+  // * Move WebTorrent to its own renderer process, similar to the way it's done in
+  //   WebTorrent Desktop
+  // * Remove this CSP exception:
+  cspDirectives['connect-src'] = '*'
```

Comment:
@dcposch only to the braveExtension, iirc. for security reasons, i would prefer that WebTorrent be its own "extension" similar to how we handle PDF reader (PDF.JS)